crates/astria-composer/src/searcher/mod.rs

```diff
@@ -204,10 +215,26 @@ impl Searcher {
                 // serialize and sign sequencer tx for incoming pending rollup txs
                 Some(rollup_tx) = self.new_transactions_rx.recv() => self.bundle_pending_tx(rollup_tx),
 
-                // submit signed sequencer txs to sequencer
+                // submit sequence actions to the executor for bundling and submission
                 Some(join_result) = self.conversion_tasks.join_next(), if !self.conversion_tasks.is_empty() => {
                     match join_result {
-                        Ok(bundle) => self.handle_bundle_execution(bundle).await,
+                        Ok(seq_action) => {
+                            let seq_action_sz = seq_action.data.len() + ROLLUP_ID_LEN;
+                            match self.seq_actions_tx.try_send(seq_action) {
+                                Ok(()) => {
+                                    // TODO: use span from the rollup transaction to log here instead
+                                    debug!(
+                                        bytes = seq_action_sz,
+                                        "bundled rollup transaction",
+                                    );
+                                }
+                                Err(e) => warn!(
+                                    error.message = %e,
+                                    error.cause_chain = ?e,
+                                    "failed to forward sequence action to the executor due to backpressure",
```

Comment:
could also be due to executor exiting

Reply:
shouldnt that trigger the branch that handles the dead executor tho?